.github/workflows/elixir.yml

```diff
@@ -0,0 +1,62 @@
+name: Elixir CI
+
+on:
```

Comment:
I am not super familiar with github actions. Is there any particular reason to limit the actions to these branches instead of just running on any `push` or `pull_request` event?

Reply:
Not really, I just added a similar workflow to my own repository and I'll probably also enable it for all branches.